examples/Image Colorbar Test.ipynb

```diff
@@ -0,0 +1,375 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For this test, we will place two widgets side-by-side."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The following line is needed to load the Firefly Javascript API into the DOM. The url can be changed to a remote instance, of course."
```

Comment:
This notebook cell should be removed -- no longer applicable.

Reply:
fixed